roles/telemetry_logging/tasks/journal_tests.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: Get journals
+  ansible.builtin.shell:
+    cmd:
+      tstamp=$(date -d '30 minute ago' "+%Y-%m-%d %H:%M:%S")
+      journalctl -t "{{ item }}" --no-pager -S "${tstamp}" | wc -l
```

Comment:
You could convert this to a `command` if you want, by removing the use of wc and making use of journal_output.stdout_lines to count the number of returned lines. This would also make debugging easier, since using verbose output would let you see what was returned from the journalctl command, rather than just seeing a number.

The `failed_when` condition could become ``journal_wc.stdout_lines | length <= 1